protobuf/flow/access/access.proto

```diff
@@ -315,6 +323,17 @@ message ProtocolStateSnapshotResponse {
   bytes serializedSnapshot = 1;
 }
 
+// Execution State
+
+message GetExecutionDataByBlockIDRequest {
+  bytes block_id = 1;
+}
+
+message GetExecutionDataByBlockIDResponse {
+  bytes block_id = 1;
```

Comment:
the execution data already has block id, so you can skip it

Reply:
Addressed!